src/logic/azext_logic/generated/_params.py

```diff
@@ -0,0 +1,132 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+
+from argcomplete.completers import FilesCompleter
+from knack.arguments import CLIArgumentType
+from azure.cli.core.commands.parameters import (
+    file_type,
+    tags_type,
+    get_enum_type,
+    resource_group_name_type,
+    get_location_type
+)
+from azext_logic.action import (
+    AddIntegrationAccount,
+    AddSku,
+    AddKeyVault
+)
+
+
+def load_arguments(self, _):
+
+    with self.argument_context('logic workflow list') as c:
+        c.argument('resource_group_name', resource_group_name_type,
+                   help='The resource group name.')
+        c.argument(
+            'top', help='The number of items to be included in the result.')
+        c.argument('filter', help='The filter to apply on the operation. Options for filters include: State, Trigger, a'
+                   'nd ReferencedResourceId.')
+
+    with self.argument_context('logic workflow show') as c:
+        c.argument('resource_group_name', resource_group_name_type,
+                   help='The resource group name.')
+        c.argument('name', options_list=[
+                   '--name', '-n'], help='The workflow name.')
+
+    with self.argument_context('logic workflow create') as c:
+        c.argument('resource_group_name', resource_group_name_type,
+                   help='The resource group name.')
+        c.argument('name', options_list=[
+                   '--name', '-n'], help='The workflow name.')
+        c.argument('definition', type=file_type, help='Path to a workflow JSON file. ' +
+                   'This JSON format should match what the logic app design tool exports', completer=FilesCompleter())
+        c.argument('location', arg_type=get_location_type(
+            self.cli_ctx), help='The resource location.')
```

Comment:
The help text could be removed. `get_location_type` has provided a more detailed help message.

Reply:
Suggest to add `validator=get_default_location_from_resource_group` for convenience.

Reply:
added